libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1219,6 +1219,7 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
         break;
 
     case MAV_CMD_DO_MOUNT_CONTROL:                      // MAV ID: 205
+        // TODO: this is only valid if packet.z == MAV_MOUNT_MODE_MAVLINK_TARGETING
```

Comment:
Should this be returning `MAV_MISSION_INVALID_PARAM7` if the mode (`packet.z`) is not set to `MAV_MOUNT_MODE_MAVLINK_TARGETING`?